plugins/net_plugin/net_plugin.cpp

```diff
@@ -623,6 +628,7 @@ namespace eosio {
       void reset() {
          fc::lock_guard g( _mtx );
          _write_queue.clear();
+         _sync_write_queue.clear();
```

Comment:
Added back in the `_sync_write_queue` removed in #1090. Had a test failure due to block ordering here causing unlinkable blocks. This is really separate from block nack. If desired I could pull it out into a different PR.